### PR TITLE
Isaac jennings signoff master control

### PR DESCRIPTION
**Pull request by iljennings**

### CODE REVIEW

File: Documentation/Signoffs/IsaacJennings-Signoff-MasterControl.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
+# Master Control Signoff
+## Functionality
+![Alt text](./Images/Master_Control_Pinout)
```
Comment (n37chen) [inline Documentation/Signoffs/IsaacJennings-Signoff-MasterControl.md]: This figure is not showing...

File: Documentation/Signoffs/IsaacJennings-Signoff-MasterControl.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+The Master Control subsytem will facilitate navigation and control of the other
+	subsystems to direct the robot around the board. The system will receive 
+	inputs from the start button, sensing microcontroller, and other peripherals
+	designed by future teams, and it will output instructions to the motor Control
+	subsystem and other specified systems (designed by future teams). The subsystem will use a 
+	Jetson Nano microcontroller to do all calculations and to send instructions. 
+	The system will ultimately perform pathfinding, navigation, and user-defined peripheral tasks,
+	but for the scope of the current system, pathfinding and navigation will be addressed 
+	and user-defined peripherals will be considered and allotted for.
```
Comment (n37chen) [inline Documentation/Signoffs/IsaacJennings-Signoff-MasterControl.md]: Is the last sentence finished? for future?

File: Documentation/Signoffs/IsaacJennings-Signoff-MasterControl.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+Data Synchronization between multiple microcontrollers will be achieved with the libraries provided on
+the Jetson Nano. Commands like stream.synchronize() will achieve the desired synchronization of data streams. 
+
+To have enough storage for any navigation algorithm that is conceivable on the Jetson Nano, a 256GB SD card will be purchased to ensure the 
+proper amount of space is available for all applications. In order to interface with the Jetson Nano, a keyboard and mouse will also be needed.
+
+## ROS Implementation and Testing
+
+The robot operating system (ROS) was used in the 2023 SECON robot as the master level controller.
```
Comment (n37chen) [inline Documentation/Signoffs/IsaacJennings-Signoff-MasterControl.md]: Please provide references here.

**Review by n37chen — CHANGES_REQUESTED** [review]

File: Documentation/Signoffs/IsaacJennings-Signoff-MasterControl.md (hunk 1 above)
Comment (iljennings) [reply Documentation/Signoffs/IsaacJennings-Signoff-MasterControl.md]: Figure should be showing now

File: Documentation/Signoffs/IsaacJennings-Signoff-MasterControl.md (hunk 2 above)
Comment (iljennings) [reply Documentation/Signoffs/IsaacJennings-Signoff-MasterControl.md]: Fixed the last sentence to make it more concise: says "allotted for future use" now

File: Documentation/Signoffs/IsaacJennings-Signoff-MasterControl.md (hunk 3 above)
Comment (iljennings) [reply Documentation/Signoffs/IsaacJennings-Signoff-MasterControl.md]: A reference to the GitHub was added to the top of the references

**Review by n37chen — APPROVED** [review]

File: Documentation/Signoffs/IsaacJennings-Signoff-MasterControl.md (hunk 1 above)
Comment (n37chen) [reply Documentation/Signoffs/IsaacJennings-Signoff-MasterControl.md]: In the text, it is still not showing...

File: Documentation/Signoffs/IsaacJennings-Signoff-MasterControl.md (hunk 3 above)
Comment (n37chen) [reply Documentation/Signoffs/IsaacJennings-Signoff-MasterControl.md]: Could you get the reference number in the description? Is it the first one?

**Review by n37chen — CHANGES_REQUESTED** [review]
I still have two comments need to be addressed.

**Review by n37chen — APPROVED** [review]

**Pull request merged**